wizard.wsgi

```diff
@@ -0,0 +1,32 @@
+"""
+<Program Name>
+  wizard.wsgi
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  April 06, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Sample WSGI file to serve Flask app with Apache, as suggested in
+  http://flask.pocoo.org/docs/0.12/deploying/mod_wsgi/
+
+  Note:
+    Expects the app's dependencies to be installed in a virtualenv
+    called `in-toto-layout` at below path.
+
+"""
+
+activate_this = "~/.virtualenvs/in-toto-layout/bin/activate_this.py"
```

Comment:
This is very specific to the way you named your venv no?

Reply:
I also don't see this script anywhere... where is it?

Reply:
Are the comments in the header block unclear? I probably should remove the entire sample script. What do you think?

Reply:
Hmm, yeah I was a little confused by the whole deal. The below path is `activate_this`? I thought the name was usually `activate` for the venv activation script...